pkg/crypto/clef/clef.go

```diff
@@ -30,7 +32,13 @@ type ExternalSignerInterface interface {
 	Accounts() []accounts.Account
 }
 
+// ExternalSignerInterface is the interface for rpc.RpcClient
+type RpcClient interface {
+	Call(result interface{}, method string, args ...interface{}) error
+}
+
 type clefSigner struct {
+	client  RpcClient // low-level rpc client to clef as ExternalSigner does not implement account_signTypedData
```

Comment:
is the comment needed?

Reply:
I would leave it because it explains why there are two different clients to the same clef.